mlrun/runtimes/nuclio/application/application.py

```diff
@@ -488,6 +489,39 @@ def invoke(
             **http_client_kwargs,
         )
 
+    @classmethod
+    def deploy_reverse_proxy_image(cls):
+        """
+        Build the reverse proxy image and save it.
+        The reverse proxy image is used to route requests to the application sidecar.
+        This is useful when you want to decrease build time by building the application image only once.
+
+        :param use_cache:   Use the cache when building the image
+        """
+        # create a function that includes only the reverse proxy, without the application
+        from mlrun import get_run_db
+        from mlrun.run import new_function
```

Comment:
Why is this import being added inside the function instead of at the top of the file? 
Also, use `import X` instead of `from X import Y`

Reply:
no actual reason, just copy-paste thing.